seed/views/v3/access_levels.py

```diff
@@ -111,7 +111,6 @@ def descendant_tree(self, request, organization_pk=None):
     @has_perm_class("requires_owner")
     @action(detail=False, methods=["POST"])
     @swagger_auto_schema(
-        manual_parameters=[AutoSchemaHelper.query_org_id_field()],
```

Comment:
org id is automatically included as it appears in the url. "api/organizations/{org_id}/add_instance"